proxy/proxy.go

```diff
@@ -145,6 +145,69 @@ type OpenTracingParams struct {
 	ExcludeTags []string
 }
 
+type PassiveHealthCheck struct {
+	// The period of time after which the endpointregistry begins to calculate endpoints statistics
```

Comment:
I think for godocs you need to start the comment with field name 
`// Period ....`